iglo/league/models.py

```diff
@@ -1019,6 +1023,17 @@ def is_delayed(self):
     @property
     def is_editable_by_player(self):
         return not self.round.is_closed() and self.group.season.state == SeasonState.IN_PROGRESS
+        
+    @property
+    def is_egd_eligible(self) -> bool:
+        """
+        Determine if a game is eligible for EGD/EGF submission based on player approvals.
+        Both players must exist (not BYE games) and have egd_approval.
+        The game doesn't need to be played yet to be marked as eligible in the UI.
+        """
+        return (self.black and self.white and 
+                self.black.player.egd_approval and 
+                self.white.player.egd_approval)
```

Comment:
what if a player plays a couple of games and changes their mind?

Reply:
Thanks for catching it.
Fixed ... I believe.